### ISSUE TITLE
Hide Code button doesn't have sufficient contrast ratio

### ISSUE THREAD

**Issue by coliff** [trimble-oss/website-modus-react-bootstrap.trimble.com#206]
Discovered via Lighthouse:
https://googlechrome.github.io/lighthouse/viewer/?psiurl=https%3A%2F%2Fmodus-react-bootstrap.trimble.com%2Fcomponents%2Flists%2F&strategy=desktop&category=accessibility&utm_source=lh-chrome-ext

![image](https://user-images.githubusercontent.com/1212885/175509061-6878151d-d521-4d9d-9f6f-ec6961a49be8.png)
